resources/views/service/edit_registration.blade.php

```diff
@@ -35,15 +36,17 @@
                                     <td><input name="carers[]" type="hidden"
                                                value="{{ $sec_carer->name }}">{{ $sec_carer->name }}</td>
                                     <td>
-                                        <button type="button" class="remove_field"><i class="fa fa-minus" aria-hidden="true"></i>
+                                        <button type="button" class="remove_field"><i class="fa fa-minus"
+                                                                                      aria-hidden="true"></i>
```

Comment:
I think your formatter got carried away.

Reply:
Yup, it's trying to obey the "line should be X long" rule the code-sniffer is enforcing. 
Will de-format a bit.